config/confighttp/README.md

```diff
@@ -26,6 +26,31 @@ README](../configtls/README.md).
 - `compression`: Compression type to use among `gzip`, `zstd`, `snappy`, `zlib`, `deflate`, and `lz4`.
   - look at the documentation for the server-side of the communication.
   - `none` will be treated as uncompressed, and any other inputs will cause an error.
+- `compression_params` : Configure advanced compression options
+  - `level`: Configure compression level for `compression` type
+  - The following are valid combinations of `compression` and `level`
+    - `gzip`
+      - NoCompression: `0`
+      - BestSpeed: `1`
+      - BestCompression: `9`
+      - DefaultCompression: `-1`
+    - `zlib`
+      - NoCompression: `0`
+      - BestSpeed: `1`
+      - BestCompression: `9`
+      - DefaultCompression: `-1`
+    - `deflate`
+      - NoCompression: `0`
+      - BestSpeed: `1`
+      - BestCompression: `9`
+      - DefaultCompression: `-1`
+    - `zstd`
+      - SpeedFastest: `1`
+      - SpeedDefault: `3`
+      - SpeedBetterCompression: `6`
+      - SpeedBestCompression: `11`
```

Comment:
Does it mean that `zstd` has more levels? I see we don't validate that

Reply:
zstd doesn't seem to need validation because it picks the nearest level to whatever number is entered.